main_test.go

```diff
@@ -1,12 +1,21 @@
 package main_test
 
 import (
+	"os"
 	"testing"
 
 	"github.com/nlnwa/warchaeology/cmd"
 )
 
 func TestEmptyCommandPrompt(t *testing.T) {
+	stdout_backup := os.Stdout
+	stderr_backup := os.Stderr
+	os.Stdout = nil
+	os.Stderr = nil
+
 	shell := cmd.NewCommand()
 	_ = shell.Execute()
+
+	os.Stdout = stdout_backup
+	os.Stderr = stderr_backup
```

Comment:
```suggestion
	os.Stdout = nil
	os.Stderr = nil

	shell := cmd.NewCommand()
	_ = shell.Execute()
```

Why not keep it simple?

Reply:
Added a fixup commit: https://github.com/nlnwa/warchaeology/pull/88/commits/0369090f782fa232584894d757318dd0f64d312d